build.sh

```diff
@@ -16,5 +27,5 @@ mkdir -p $P/scratch;
 cd $P/scratch;
 rm -rf $P/install;
 $SRC/configure --prefix=$P/install --with-mountutildir=$P/install/sbin --with-initdir=$P/install/etc --localstatedir=/var --enable-bd-xlator=yes --enable-debug --silent
-make install CFLAGS="-g -O0 -Wall -Werror" -j 4 >/dev/null
+make install CFLAGS="-g -O0 -Wall -Werror" -j ${nproc} >/dev/null
```

Comment:
Are you saying that you think we should just use -j without an argument?  That's fine - the platform check is still useful for other things - but it wasn't clear whether that's what you meant.